.storybook/main.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  "stories": [
+    "../src/keycloak-theme/pages/Login.stories.tsx",
```

Comment:
for some reason, the usual 'glob' approach `../src/**/*stories.tsx` did not work for me, need to investigate  this

Reply:
weird powershell bug. resolved